ajax/libs/indexeddbshim/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "indexeddbshim",
+  "filename": "indexeddbshim.min.js",
+  "version": "2.2.1",
+  "author": "Parashuram <code@nparashuram.com>",
+  "description": "A polyfill for IndexedDB using WebSql",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/axemclion/IndexedDBShim.git"
+  },
+  "keywords": [
+    "indexedDB",
+    "database",
+    "polyfill",
+    "websql"
+  ],
+  "license": "Apache-2.0 or MIT",
```

Comment:
thanks @brettz9, we use SPDX expressions, too. Just keep the same structure when we meet multi-license lib in cdnjs.